tutoraspects/commands_v1.py

```diff
@@ -27,7 +27,7 @@ def load_xapi_test_data(config_file) -> list[tuple[str, str]]:
     return [
         (
             "aspects",
-            "echo 'Running script...' && "
+            f"echo 'Running script... {config_file}' && "
```

Comment:
The xapi-db-load changes are unrelated, but are useful for debugging what's actually being run there.